src/components/sidebar/Menu.js

```diff
@@ -22,22 +21,29 @@ const Menu = ({ theme, topHeight, transition }) => {
       </header>
       <MenuMap
         header="Sorting Algorithms"
-        items={SORTING_ALGOS} />
+        items={SORTING_ALGOS}
+        name= "SORTING_ALGOS"
+      />
       <MenuMap
         header="Data Structures"
-        items={DATA_STRUCTURES} />
+        items={DATA_STRUCTURES}
+        name="DATA_STRUCTURES"
+      />
       <details open>
         <summary className={`sidebar--menu--sub-header ${theme}`}>
           Algorithm Challenges
         </summary>
         <MenuMap
           header="Easy"
           items={EASY_ALGOS}
-          xtraClass="sub" />
+          name= 'EASY_ALGOS'
+        />
         <MenuMap
           header="Moderate"
+          xtraClass="sub"
           items={MODERATE_ALGOS}
-          xtraClass="sub" />
+          name= 'MODERATE_ALGOS'
```

Comment:
let's keep the spacing after `=` consistent in this component, i.e. `name="MODERATE_ALGOS"`

Reply:
Done